src/api/route-services/plots-tables.js

```diff
@@ -28,6 +28,28 @@ class PlotsTablesService {
     return tableData;
   }
 
+  async updatePlotData(experimentId, plotUuid, plotData) {
+    const marshalledData = convertToDynamoDbRecord({ ':d': plotData });
+
+    logger.log('debugging marshalled data', marshalledData);
+
+    const params = {
+      TableName: this.tableName,
+      Key: {
+        experimentId: { S: experimentId }, plotUuid: { S: plotUuid },
+      },
+      UpdateExpression: 'SET plotData = :d',
+      ExpressionAttributeValues: marshalledData,
+      ReturnValues: 'UPDATED_NEW',
```

Comment:
can you tell me what will the structure of the plot information be like? If you can paste me an example plot saved in DynamoDB just to help myself viusualise it

Reply:
Also, just to double check -- here you are only updating the `data` part of the plot information in Dynamo? You are not touching the other parts like the config with the styling settings or the configuration settings?

Reply:
Yes, only updating the plot data, not touching the other parts of the plot information

Reply:
Using test values

```
output.plotData = [
    {
      x: 1,
      y: 2,
      values: 3,
    },
    {
      x: 2,
      y: 3,
      values: 4,
    },
    {
      x: 3,
      y: 4,
      values: 5,
    },
  ];
```

This is the DynamoDB record that's stored for `genesVsUmisScatterplot` : 

```
{
    "plotUuid": {
        "S": "genesVsUmisScatterplot"
    },
    "experimentId": {
        "S": "e52b39624588791a7889e39c617f669e"
    },
    "plotData": {
        "L": [
            {
                "M": {
                    "x": {
                        "N": "1"
                    },
                    "y": {
                        "N": "2"
                    },
                    "values": {
                        "N": "3"
                    }
                }
            },
            {
                "M": {
                    "x": {
                        "N": "2"
                    },
                    "y": {
                        "N": "3"
                    },
                    "values": {
                        "N": "4"
                    }
                }
            },
            {
                "M": {
                    "x": {
                        "N": "3"
                    },
                    "y": {
                        "N": "4"
                    },
                    "values": {
                        "N": "5"
                    }
                }
            }
        ]
    }
}

```

Reply:
thanks! I was asking more about the structure you described in here: https://github.com/biomage-ltd/api/pull/61#discussion_r592454902 but anyway, it looks good!